Sources/SwiftDocC/Model/MarkdownOutput/Translation/MarkdownOutputSemanticVisitor.swift

```diff
@@ -187,21 +186,35 @@ extension MarkdownOutputSemanticVisitor {
             for destination in relationshipGroup.destinations {
                 switch context.resolve(destination, in: identifier) {
                 case .success(let resolved):
-                    add(target: resolved, type: .relatedSymbol, subtype: relationshipGroup.kind.rawValue)
+                    add(target: resolved, type: .relatedSymbol, subtype: relationshipGroup.kind.manifestRelationship)
                 case .failure:
                     if let fallback = symbol.relationships.targetFallbacks[destination] {
-                        add(fallbackTarget: fallback, type: .relatedSymbol, subtype: relationshipGroup.kind.rawValue)
+                        add(fallbackTarget: fallback, type: .relatedSymbol, subtype: relationshipGroup.kind.manifestRelationship)
                     }
                 }
             }
         }
+        // TODO: add support for missing sections rdar://166124742
         return MarkdownOutputNode(metadata: metadata, markdown: markdownWalker.markdown)
+        
     }
 }
 
 import SymbolKit
 
-extension MarkdownOutputNode.Metadata.Symbol {
+private extension RelationshipsGroup.Kind {
+    var manifestRelationship: MarkdownOutputManifest.RelationshipSubType? {
+        // Structured like this to cause a compiler error if a new case is added
+        switch self {
+        case .conformingTypes: .conformingTypes
+        case .conformsTo: .conformsTo
+        case .inheritsFrom: .inheritsFrom
+        case .inheritedBy: .inheritedBy
+        }
+    }
```

Comment:
Question: what's the goal of this code? Should this enumeration be defined so that new callers need to handle unknown cases?